flopy/mbase.py

```diff
@@ -1242,18 +1241,17 @@ def set_version(self, version):
 
     def change_model_ws(
         self,
-        new_pth: Optional[Union[str, os.PathLike]] = os.curdir,
+        new_pth: Union[str, PathLike] = curdir,
```

Comment:
For this case, `None` was not added to `Union`, since the default was `curdir` (as it should be). Docstring below is adjusted to remove mention of `None`.